src/Tools/dotnet-monitor/Experimental/ExperimentalFlags.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+#if UNITTEST
+namespace Microsoft.Diagnostics.Monitoring.TestCommon
+#else
+namespace Microsoft.Diagnostics.Tools.Monitor
+#endif
+{
+    internal static class ExperimentalFlags
+    {
+        private const string ExperimentalPrefix = "DotnetMonitor_Experimental_";
+
+        // Feature flags
+        public const string Feature_CallStacks = ExperimentalPrefix + nameof(Feature_CallStacks);
```

Comment:
In the same vein as the prior comment (so this may not be applicable) - if we're using double underscores to separate, then should this be "Feature__CallStacks"?